<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PGIntention</class>
 <widget class="QWidget" name="PGIntention">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="tableIndWonder">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>481</width>
     <height>421</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>名称</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>书单</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="tableCorWonder">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>370</y>
     <width>481</width>
     <height>421</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>名称</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>书单</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="tableMembers">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>271</width>
     <height>131</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>名称</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>会员类型</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="lineMemSearch">
   <property name="geometry">
    <rect>
     <x>232</x>
     <y>10</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="PBMemSearch">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PBSelect">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>10</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboMemBox">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>72</width>
     <height>23</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Individual</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Corporate</string>
    </property>
   </item>
  </widget>
  <widget class="QTableWidget" name="tableBookList">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>10</y>
     <width>331</width>
     <height>141</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>书名</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>价格</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>数量</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="tableInven">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>190</y>
     <width>331</width>
     <height>161</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>书名</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>价格</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>数量</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="PBBookSearch">
   <property name="geometry">
    <rect>
     <x>589</x>
     <y>160</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineBookSearch">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>160</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineAmount">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>160</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="PBBookAdd">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>160</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PBBookRemove">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>160</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PBSearch">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>330</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineSearch">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>330</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboSearchBox">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>330</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Individual</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Corporate</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="PBRemove">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>330</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PBRefreshMems">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>40</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh members</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PBSave">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PBImport">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineVersion">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>330</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>选择一个版本</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PBAdd">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>260</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>ADD</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PBRefreshInven">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>200</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh inventory</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PBRefreshIntention">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>260</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh intention</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PBRefresh">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh ALL</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PBRefreshList">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>110</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh BookList</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt; font-weight:700;&quot;&gt;购书意向&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
